Sources/BraveWallet/Crypto/Stores/NFTStore.swift

```diff
@@ -153,6 +153,8 @@ public class NFTStore: ObservableObject, WalletObserverStore {
   private var metadataCache: [String: NFTMetadata] = [:]
   /// Spam from SimpleHash in form of `NetworkAssets`
   private var simpleHashSpamNFTs: [NetworkAssets] = []
+  /// All User NFTs that includes visible, hidden, spam
+  private var allUserNFTs: [BraveWallet.BlockchainToken] = []
```

Comment:
I think this could just be a variable in the `update()` function? Not used elsewhere